plugins/azure-devops/src/api/AzureDevOpsClient.ts

```diff
@@ -42,6 +48,17 @@ export class AzureDevOpsClient implements AzureDevOpsApi {
     return { items };
   }
 
+  async getPullRequests(
```

Comment:
```suggestion
  public async getPullRequests(
```

Reply:
I've added this here and to the `getRepoBuilds` above to keep this consistent